### ISSUE TITLE
Grab Alerts Tested - a few issues found

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#366]
March 12th, testing BASE from PhET Test
Voice Over and Safari (11.03) on Mac OS 10.13.3
I tested the Grab Alerts in all contexts:
- single-balloon scenarios: In Play Area, On Sweater, At wall, While independently moving in all charge views
- two-balloon scenarios, I tested all contexts again when next to or not next to the other balloon.

Here are the very few issues I found fro the Grab Alerts:
**Show all charges**
- [x] the combined induced charge description is only applied to the single grabbed balloon when a balloon is grabbed at wall while next to the other charged balloon. Induced charge description should contain the parameter "balloons" as it does in the dragging alert.

**Punctuation (Show all charges)**
 - [x] Grab Alert on Sweater, the Sweater description has an extra period at the end.

**Show charge differences**
- [x] Grab Alert on sweater (two-balloons): incorrect positive charge count for sweater. The described amount only reflects the grabbed balloon's negative charge count. The positive count has to combine all unpaired positive charges on the sweater.
 
**Punctuation (Show charge differences)**
- [x] Grab Alert on sweater, missing period at the end of balloon description.
- [x] Grab Alert at wall (two balloons), missing period at the end of balloon description and at end of wall description.

**Show no charges**
I didn't notice any Grab Alert issues with "Show no charges" selected.

**Comment by terracoda** [comment]
@jessegreenberg, I didn't notice at first - just realized that [+ Balloon visual charge (with Each)] phrase and the related special cases when balloons are next-to each other are not yet implemented.

The combine induced charge effect for "Show all charges" is implemented and sounds great. I will add [+ Balloon visual charge (with Each)]  to the list in the previous issue.

**Comment by terracoda** [comment]
@jessegreenberg, there is a separate issue related to [+ Balloon visual charge (with Each)] , so just mentioning it here in this issue rather than adding it to the list of issues in the first comment.

**Comment by jessegreenberg** [comment]
The  grab alerts are mostly done - but the alerts are incorrect when balloons have differing charges. I need to refactor BalloonDescriber to make it possible to get a decscription of the other balloons relative charge.

**Comment by jessegreenberg** [comment]
Done in the following commit. Now I need to replace all usages of `this.getRelativeChargeDescriptionWithLabel` with `BalloonDescriber.getRelativeChargeDescriptionWithLabel`.

**Comment by jessegreenberg** [comment]
Also for `getRelativeChargeDescription`, which had to be moved to statics.

**Comment by jessegreenberg** [comment]
Changes looking good, but refactor made balloon labels get added to single single balloon grab alerts.

**Comment by jessegreenberg** [comment]
> Grab Alert on sweater (two-balloons): incorrect positive charge count for sweater. The described amount only reflects the grabbed balloon's negative charge count. The positive count has to combine all unpaired positive charges on the sweater.

I think this must have been fixed through other work, I am getting the correct charge descriptino now.

**Comment by jessegreenberg** [comment]
Alright, we are there for this issue. Thanks @terracoda!
